package.json

```diff
@@ -13,6 +13,8 @@
   },
   "scripts": {
     "start": "node src/index.js",
+    "build": "npm run clean && cp -r configs dist/configs && cp -r src dist/src && cp -r public dist/public && cp *.json dist && rimraf dist/package-lock.json",
```

Comment:
We don't need `public` folder. It's only for testing purpose.

Reply:
Not sure about `rimraf dist/package-lock.json`. I think we can keep package-lock.json together with package.json, so when packaging, we can use exact same versions defined in package-lock.json.

Reply:
ok, keeping this

Reply:
added Readme.md too

Reply:
removed public folder too